rest/api.go

```diff
@@ -199,14 +199,20 @@ func (h *handler) handleGetDB() error {
 		lastSeq, _ = h.db.LastSequence()
 	}
 
+	compactRunning := false
+
+	if atomic.LoadUint32(&h.db.CompactState) == db.DBCompactRunning {
```

Comment:
As discussed, move to a `db.CompactRunning` method that returns a bool and call directly from struct below.